queries/java/highlights.scm

```diff
@@ -150,25 +167,30 @@
 "case"
 ] @conditional
 
-; 
+;
```

Comment:
Why keeping this ?

Reply:
To separate sections, I don't know what to name that section